db/structure.sql

```diff
@@ -2534,13 +2533,34 @@ CREATE INDEX index_recents_on_classification_id ON recents USING btree (classifi
 
 
 --
+-- Name: index_recents_on_project_id; Type: INDEX; Schema: public; Owner: -; Tablespace: 
+--
+
+CREATE INDEX index_recents_on_project_id ON recents USING btree (project_id);
```

Comment:
That's a pretty good question. The migration is super simple and wouldn't have added those. Guess I'll remove them?

```class RemoveNameFromOrganization < ActiveRecord::Migration
  def change
    remove_column :organizations, :name
  end
end```